app/views/layouts/application.html.erb

```diff
@@ -16,21 +16,8 @@
         <h1>Little Esty Shop</h1>
       </div>
     </header>
```

Comment:
Move github API information footer to a partial. Can put it on any page I want to incorporate it on.